GNUmakefile

```diff
@@ -77,10 +82,29 @@ teamcity-servicemap-generate:
 	cd tools/teamcity-generator;\
 		go run . --output $(OUTPUT_PATH) --version $(VERSION)
 
+
+clean-provider:
+	cd $(OUTPUT_PATH);\
+		go mod download;\
+		find . -type f -not -wholename "./.git*" -not -wholename "./.changelog*" -not -name ".travis.yml" -not -name ".golangci.yml" -not -name "CHANGELOG.md" -not -name "CHANGELOG_v*.md" -not -name "GNUmakefile" -not -name "docscheck.sh" -not -name "LICENSE" -not -name "README.md" -not -wholename "./examples*" -not -name ".go-version" -not -name ".hashibot.hcl" -print0 | xargs -0 git rm > /dev/null
+
+clean-tgc:
+	cd $(OUTPUT_PATH);\
+		rm -rf ./tfplan2cai/testdata/templates/;\
+		rm -rf ./tfplan2cai/testdata/generatedconvert/;\
+		rm -rf ./tfplan2cai/converters/google/provider;\
+		rm -rf ./tfplan2cai/converters/google/resources;\
+		find ./tfplan2cai/test/** -type f -exec git rm {} \; > /dev/null;\
+
 tgc:
 	cd mmv1;\
 		bundle;\
-		bundle exec compiler -e terraform -f tgc -v beta -o $(OUTPUT_PATH) $(mmv1_compile);\
+		bundle exec compiler -e terraform -f tgc -v beta -o $(OUTPUT_PATH)/tfplan2cai $(mmv1_compile);\
```

Comment:
technically we're changing the behavior (output path) of `make tgc`. I'm sure if there's anything prevent us from doing so though, like if we try not to break users

Reply:
@melinath could you take a look at Shuya's comments regarding tgc compilation. Are the changes we're making okay from your POV?

Reply:
I think this change is fine - it makes sure that we support generating both parts of TGC, and we don't have many folks relying on it (attn @iyabchen @amirkaromashkin @edisonhuang-google FYI). You'll need to update the documentation here: https://github.com/GoogleCloudPlatform/terraform-google-conversion/blob/main/docs/contributing/add_new_resource.md

Reply:
oh - you actually still need to add the cai2hcl line here, though.

Reply:
added